### PR TITLE
[Enhancement](regression-test)Add regression test for MoW backup and restore

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

File: regression-test/suites/backup_restore/test_MoW_backup_restore.groovy [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    "disable_auto_compaction" = "true",
+    "enable_unique_key_merge_on_write" = "true");"""
+
+    // version1 (1,1)(2,2)
+    sql """insert into ${tableName} values(1,1),(2,2)"""
+    sql """backup snapshot ${context.dbName}.snapshot1 to ${repo} on (${tableName}) properties("type"="full")"""
+    while(checkSnapshotFinish()==false){
+        Thread.sleep(3000)
+    }
+    qt_3 """select * from ${tableName}"""
```
Comment (zhannngchen) [inline regression-test/suites/backup_restore/test_MoW_backup_restore.groovy:41]: All select should add `order by`, to make your query result stable

File: regression-test/suites/backup_restore/test_MoW_backup_restore.groovy [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    }
+    qt_13 """select * from ${tableName}"""
+    // version4 (2,2)
+    assertTrue(syncer.getSnapshot("snapshot4", "${tableName}"))
+    assertTrue(syncer.restoreSnapshot())
+    while (checkRestoreFinish() == false) {
+        Thread.sleep(3000)
+    }
+    qt_14 """select * from ${tableName}"""
+}
```
Comment (zhannngchen) [inline regression-test/suites/backup_restore/test_MoW_backup_restore.groovy:136]: it's required an extra line at the end of file

**Comment by Yukang-Lian** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**